extract_from_hg.py

```diff
@@ -195,15 +196,110 @@ def commitCset(cset):
     runCommand(['git', 'commit', '-m', commitMsg, '--author=' + cset.user(),
                 '--date=' + str(csetDate)])
 
+def formatCommitMessageLine(line):
+    # Now do translations on the line.
+
```

Comment:
To be honest, I think this would be more readable and maintainable if you extracted the relevant information into individual variables and then created a new string (eg using print and the % operator) rather than using regexps to edit the string in-place.  That said, this is not a huge function, so it wouldn't be the end of the world leaving it as is.